### PR TITLE
[BPK-1001] Update Android text tokens

### PR DESCRIPTION
**Pull request by matteo-hertel**
I've also updated the docs page to include Android text screenshot, it looks like this
[Updated Screenshot]
screen shot 2017-10-09 at 16 34 55-fullpage

need to be validated from the design pov cc @jamesf3rguson

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -9,7 +9,7 @@
   - iOS touchable highlight underlay tokens
   - iOS shadow tokens
   - iOS & Android radii tokens
-
+  - Update Android text tokens
```
Comment (matthewdavidson) [inline changelog.md]: Massive nitpick: 

Can we have a newline between components in the changelog? 😇

File: native/packages/react-native-bpk-component-text/src/BpkText-test.ios.js [hunk 2]
```diff
@@ -16,18 +16,25 @@
...
 
+import BpkText from './BpkText';
 import commonTests from './BpkText-test.common';
 
 jest.mock('react-native', () => {
   const reactNative = require.requireActual('react-native');
   jest
-      .spyOn(reactNative.Platform, 'select')
-      .mockImplementation(obj => obj.ios || obj.default);
+    .spyOn(reactNative.Platform, 'select')
```
Comment (matthewdavidson) [inline native/packages/react-native-bpk-component-text/src/BpkText-test.ios.js]: why remove these? :P

Comment (matthewdavidson) [reply native/packages/react-native-bpk-component-text/src/BpkText-test.ios.js]: We really need to integrate prettier to eradicate these sorts of stylistic nitpicks.

File: native/packages/react-native-bpk-component-text/src/BpkText.js [hunk 3]
```diff
@@ -54,36 +54,39 @@ const stylePropType = (props, propName, componentName) => {
...
   xxl: { ...getStyleByTextStyle('xxl') },
-  emphasize: { fontWeight: '600' },
 });
 
 const BpkText = (props) => {
   const { children, textStyle, style, emphasize, ...rest } = props;
-
-  return <Text style={[styles[textStyle], emphasize && styles.emphasize, style]} {...rest}>{children}</Text>;
+  const finalStyle = [styles[textStyle]];
+  // emphasize
```
Comment (matthewdavidson) [inline native/packages/react-native-bpk-component-text/src/BpkText.js]: Remove extraneous comment.

File: native/packages/react-native-bpk-component-button/src/__snapshots__/BpkButton-test.ios.js.snap [hunk 4]
```diff
@@ -90,6 +90,7 @@ exports[`iOS BpkButton should render correctly 1`] = `
               "lineHeight": 18,
             },
             Object {
+              "fontFamily": "System",
```
Comment (matthewdavidson) [inline native/packages/react-native-bpk-component-button/src/__snapshots__/BpkButton-test.ios.js.snap]: This seems extraneous, similar to https://github.com/Skyscanner/backpack/pull/177#discussion_r143409735 but a little bit more prevalent - in iOS `"fontFamily": "System",` is already being applied - seems like we can omit the `fontFamily` prop from the emphasize styles if it's iOS?

File: native/packages/react-native-bpk-component-text/src/BpkText-test.ios.js (hunk 2 above)
Comment (matteo-hertel) [reply native/packages/react-native-bpk-component-text/src/BpkText-test.ios.js]: I've got the auto-formatter on so it got removed automatically to comply with the settings in root of the project (I hope)

**Pull request merged**